models/engine/file_storage.py

```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """get the obejects by id, using dict's get()"""
```

Comment:
Typo in "objects," and feels a bit vague to me. I won't be too pedantic about docstrings, and want to compromise. But you will find that mine tend to be a bit more verbose, and follow the Google style guide that they gave us a while back:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
do we change the other method comments?